src/admin-views/tribe-options-display.php

```diff
@@ -332,6 +332,27 @@ static function( $view ) {
 			'11:00' => date_i18n( $site_time_format, strtotime( '11:00 am' ) ),
 		],
 	],
+
+	'remove_event_end_time'             => [
+		'type'            => 'checkbox_list',
+		'label'           => esc_html__( 'Remove event end time', 'tec-labs-tec-tweaks' ),
+		'tooltip'         => esc_html__(
+			                     'When this box is checked the end time will no longer display for events that end on the same day when viewing the List and Day views, the recent past events list, the tooltip in Month and Week (Pro) views, as well as on the event page itself.',
+			                     'tec-labs-tec-tweaks'
+		                     ) . '<br><i>' . esc_html__(
+			                     'Source:',
+			                     'tec-labs-tec-tweaks'
+		                     ) . ' <a href="https://theeventscalendar.com/knowledgebase/k/remove-the-event-end-time-in-views/" target="_blank">Remove the Event End Time in Views</a></i>',
+		'options'         => [
+			'recent' => esc_html__( 'Recent past events list', 'tec-labs-tec-tweaks' ),
+			'single-event' => esc_html__( 'Single event page', 'tec-labs-tec-tweaks' ),
+			'day'    => esc_html__( 'Day view', 'tec-labs-tec-tweaks' ),
+			'list'   => esc_html__( 'List view', 'tec-labs-tec-tweaks' ),
+			'month'  => esc_html__( 'Month view tooltip', 'tec-labs-tec-tweaks' ),
+		],
+		'validation_type' => 'options_multi',
+		'can_be_empty'    => true,
+	],
```

Comment:
Ideally all of this data would come from the actual settings field built by the visibility options abstract.

Reply:
IMHO Ideally in a standalone settings handler, not mixed with the visibility filter. I don't believe it should be [responsible to do multiple things](https://en.wikipedia.org/wiki/Single_responsibility_principle). Especially since we will still need settings to be handled in circumstances that the Visibility Modifier won't be necessary. 

That said, I think we can make this a lot better for sure. Thoughts?

Reply:
I agree with you that ideally we would have a standalone Settings handler, the problem is that we wont be tackling that project any time soon due to a couple of other priorities that we will need to focus on.

The reason why I had suggested the use of the abstract on ECP was to avoid rework and having to define the options and option name in two places it improves how the reliability of the settings and how these settings are used.

Later when we go about refactoring the Settings framework we can just point those to the new place.

Reply:
Made some updates. Are you asking me to refactor to put the settings in the View Modifier class?

Reply:
;)